src/static/css/pad.css

```diff
@@ -1320,22 +1324,11 @@ input[type=checkbox] {
 
 }
 
-[data-icon]:before {
-  font-family: "fontawesome-etherpad" !important;
-  content: attr(data-icon);
-  font-style: normal !important;
-  font-weight: normal !important;
-  font-variant: normal !important;
-  text-transform: none !important;
-  speak: none;
-  line-height: 1;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
+[data-icon]:before,
 [class^="icon-"]:before,
 [class*=" icon-"]:before {
   font-family: "fontawesome-etherpad" !important;
+  content: attr(data-icon);
```

Comment:
This is change was part of your original series, and I have extracted it in 1938465e6144.
Could you please explain it?

I like the grouping, but failed to understand why "attr(data-icon)" was added to two more classes.

Reply:
Don't remember exactly, but this is for handle icon like 
```html
<span class="icon-table"></span>
/* OR */
<span data-icon="table"></span>
```
So `content: attr(data-icon);` should not go with rules b) [class^="icon-"]:before

So you can group rule a) and b) without `content: attr(data-icon);` and then add new rule
```css
[data-icon]:before { content: attr(data-icon); }
```

Reply:
Could you please commit a fix on top of this PR?
I will take care of folding it.

Reply:
Done ! Finally I did some test, and this `content: attr(data-icon);` is not used and does not work, so I just removed it, everything work fine

Reply:
Deployed in beta.

Reply:
I have a question now: grepping through the code, the only remaining reference to `data-icon` is here:
https://github.com/seballot/etherpad-lite/blob/cbb4aaa15a89eee552e13f58ef02ca3cbd175592/src/static/css/pad.css#L1327-L1339

What should we do with that? Remove only line 1327? Remove lines 1327-1339?

In your evaluation, please have a look at the code that introduced those new rules, 9aea689438d3, maybe it will help making sense why they where there in the first place.

Reply:
It is used in some plugins, like ep_tables2
In order to not change each plugin, sometimes I needed to do things like that : introducing a rule only for a plugin. But actually now I'm the owner of ep_table, so if you want you can remove that line (only 1327) and I will change ep_tables for my needs !
Btw, sorry about all those messy commits, it's the first project I contribute where someone pay as much attention to commits. In my next contributions I'll make them cleaner !!